lib/dashboard.dart

```diff
@@ -0,0 +1,201 @@
+import 'package:flutter/material.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_core/firebase_core.dart';
+
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await Firebase.initializeApp();
```

Comment:
@harryalloyd Where is the firebase auth file?

Reply:
@harryalloyd Use the auth File we already have: https://github.com/sunami09/WebGroup/blob/main/android/app/google-services.json